src/somd2/config/_config.py

```diff
@@ -913,7 +915,6 @@ def output_directory(self):
 
     @output_directory.setter
     def output_directory(self, output_directory):
-        self.cirectory_existed = False
```

Comment:
Sorry, I now see that the code _is_ using the `directory_existed` attribute. This means that you will need to set it to a default value, so this typo should probably be changed to `self.directory_existed = False`. Since this isn't a property, i.e. it's just something used internally, then it's best to make it hidden so that the user doesn't change it by mistake, i.e. use `self._directory_existed`.

One problem is that this attribute is used in the `restart` setter, which would fail if the `output_directory` wasn't set first. As such, you probably want to set the default in the `Config` constructor and ensure that the `output_directory` property is set _before_ the `restart` one.

Reply:
Yes, that's a better approach. It's always cleaner when attributes are fully decoupled, otherwise you can run into self-consistency issues. Since things are validated in the runner, just remove all uses of `directory_existed`.